quipucords/scanner/network/runner/roles/cpu/tasks/main.yml

```diff
@@ -45,7 +45,7 @@
   ignore_errors: yes
 
 - name: gather internal_cpu_model_ver fact
-  raw: cat /proc/cpuinfo 2>/dev/null | grep '^model\s*.' | sed -n -e 's/^.*model\s*.\s*//p'
+  raw: cat /proc/cpuinfo 2>/dev/null | grep '^model\s*:' | sed -n -e 's/^.*model\s*:\s*//p'
```

Comment:
Why did this regex change? Well, if you look at what the old command normally produces:

```
$ cat /proc/cpuinfo 2>/dev/null | grep '^model\s*.' | sed -n -e 's/^.*model\s*.\s*//p'
63
ame	: Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz
```
See how there are TWO lines in the output? **This is a bug in the original implementation**, but we rarely (but probably _not never_) noticed it because normally we would just grab the first line of output, resulting in the value `63`. Order is not guaranteed, though, and it's possible that some reports out in the wild contained the wrong line with the model name instead of the model ID.

Remove the `sed`, and you can see more of the original output:

```
$ cat /proc/cpuinfo 2>/dev/null | grep '^model\s*.'
model		: 63
model name	: Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz
```

Now that we are grabbing the _last_ line of output for commands like this, it's more important that this command only outputs the one line that is actually relevant. After changing the regex, the new output looks like:

```
$ cat /proc/cpuinfo 2>/dev/null | grep '^model\s*:' | sed -n -e 's/^.*model\s*:\s*//p'
63
```

Much better.